Modules/System/Tax Management/src/app.json

```diff
@@ -0,0 +1,31 @@
+{
+    "id":  "93f93dda-3177-4de6-9f6e-b38103809c6d",
+    "name":  "Tax Management",
```

Comment:
I think this name is too generic, as this only works in the Netherlands. Let's work on the name. Dutch Tax Management? But this isn't really tax managment either, is it? It's the Digipoort submit part. Maybe Dutch Tax Submission? Naming discussions can be fun :-D

Reply:
Then we might as well just call it Digipoort? ;) I thought you might want to re-use it for other countries.